lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -383,6 +389,17 @@ class ReportUIFeatures {
 
   closeToolsDropdown() {
     this.toolsButton.classList.remove('active');
+    this.toolsButton.setAttribute('aria-expanded', 'false');
+  }
+
+  openToolsDropdown() {
+    this.toolsButton.classList.add('active');
+    this.toolsButton.setAttribute('aria-expanded', 'true');
+    this._document.addEventListener('keydown', this.onKeyDown);
```

Comment:
maybe we can move the `removeEventListener` part into `closeToolsDropdown` now for clarity?

we really have some cleanup to do for our `keydown` `keyup` listeners, huh? :)